racingcar-java/src/main/java/RacingCar.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingCar {
+
+    List<Car> cars;
+
+    public RacingCar(int numberCars) {
+        this.cars = setCars(numberCars);
+    }
+
+    List<Car> setCars(int numberCar) {
+        List<Car> cars = new ArrayList<Car>();
+        for (int i = 0; i < numberCar; i++) {
+            cars.add(new Car());
+        }
+        return cars;
+    }
+
+    List<Car> getCars() {
+        return cars;
+    }
+
+    int getRandom() {
+        return new Random().nextInt(10);
+    }
+
+    void playOneRound(List<Car> cars) {
+        for (Car c: cars) {
+            c.move(getRandom());
+        }
+    }
```

Comment:
```suggestion
    void playOneRound() {
        for (Car c: cars) {
            c.move(getRandom());
        }
    }
```
지금 playOnRound는 RacingCar라는 객체가 가진 Car객체들을 모두 움직이는 메소드에요.
RacingCar객체는 Car 객체 컬랙션을 가진 객체구요.

근데 지금과 같이 cars를 파라미터로 받을 필요가 있을까요? RacingCar는 cars 필드값을 갖고있지않나요?